metricbeat/docs/modules/vsphere.asciidoc

```diff
@@ -36,15 +101,24 @@ metricbeat.modules:
 - module: vsphere
   enabled: true
   metricsets: ["cluster", "datastore", "datastorecluster", "host", "network", "resourcepool", "virtualmachine"]
-  # Real-time data collection – An ESXi Server collects data for each performance counter every 20 seconds.
+  
+  # Real-time data collection – An ESXi Server collects data for each performance counter every 20 seconds by default.
+  # Supported Periods:
+  # The Datastore and Host metricsets support performance data collection using the vSphere performance API.
```

Comment:
shall we also mention that this will not impact the metrics collection other than perf metrics.

Reply:
Nice idea. let me do that.